Checkout/Checkout/Source/Model/Country.swift

```diff
@@ -277,6 +277,15 @@ public struct Country: Equatable {
     ]
   }
 
+    public init?(dialingCode: String) {
+        var dialingCode = dialingCode.components(separatedBy: .decimalDigits.inverted).joined()
+        if let country = Self.allAvailable.first(where: { $0.dialingCode == dialingCode }) {
+            self = country
+        } else {
+            return nil
+        }
```

Comment:
```suggestion
        return Self.allAvailable.first(where: { $0.dialingCode == dialingCode })
```

Reply:
I tried and i got this error message "'nil' is the only return value permitted in an initializer"